man/binfmt.d.xml

```diff
@@ -23,9 +23,10 @@
   </refnamediv>
 
   <refsynopsisdiv>
-    <para><filename>/etc/binfmt.d/*.conf</filename></para>
-    <para><filename>/run/binfmt.d/*.conf</filename></para>
-    <para><filename>/usr/lib/binfmt.d/*.conf</filename></para>
+    <para><literallayout><filename>/etc/binfmt.d/*.conf</filename>
```

Comment:
I think that, markup-wise, it would be appropriate to use `<simplelist>` for this. The default rendering of it is single-column vertical, hence it should look pretty much the same way like the `<literallayout>`.
```
<para><simplelist>
    <member><filename>/etc/binfmt.d/*.conf</filename></member>
    <member><filename>/run/binfmt.d/*.conf</filename></member>
    <member><filename>/usr/lib/binfmt.d/*.conf</filename></member>
</simplelist></para>
```

Reply:
Thanks!

Updated to use `<simplelist>`.